metadata/metadata_test.go

```diff
@@ -349,6 +349,8 @@ func TestStringerMD(t *testing.T) {
 		{MD{"k1": []string{"v1", "v2"}}, []string{"MD{k1=[v1, v2]}"}},
 		{MD{"k1": []string{"v1"}}, []string{"MD{k1=[v1]}"}},
 		{MD{"k1": []string{"v1", "v2"}, "k2": []string{}, "k3": []string{"1", "2", "3"}}, []string{"MD{", "k1=[v1, v2]", "k2=[]", "k3=[1, 2, 3]", "}"}},
+		{MD{"k2": []string{}, "k3": []string{"1", "2", "3"}, "k1": []string{"v1", "v2"}}, []string{"MD{", "k1=[v1, v2]", "k2=[]", "k3=[1, 2, 3]", "}"}},
```

Comment:
The `want` field of the table should be a `string` and not a `[]string`. The latter was supposedly used to get around the non-deterministic nature of the previous implementation.

Reply:
Done!